### PR TITLE
[Meta] Update README with new contribution guide

### PR DESCRIPTION
**Pull request by randomicecube**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,9 +26,16 @@ Abaixo encontram-se guias para instalação em Windows e Linux.
...
+- Por `https`:
+
+  ```bash
+  git clone https://github.com/diogotcorreia/resumos-leic.git
+  ```
+
+- Por `ssh`:
+  ```bash
+  git clone git@github.com:diogotcorreia/resumos-leic.git
+  ```
```
Comment (diogotcorreia) [inline README.md:38]: I don't think this is needed, only contributors have write access, therefore HTTPS is always enough for everyone else

File: README.md [hunk 2]
```diff
@@ -39,8 +46,19 @@ yarn               # or yarn install
 
 ### Alterar conteúdos
 
-Os ficheiros `.md` (markdown) estão na pasta `src`.
-Para alterar os links na sidebar, é necessário alterar o ficheiro `src/.vuepress/config.js`.
+Os ficheiros `.md` (markdown) estão na pasta da respetiva UC dentro de `content`.
+Assim que um ficheiro `.md` é adicionado à respetiva pasta da UC, é acessível a partir da sidebar.
```
Comment (diogotcorreia) [inline README.md:50]: This is not true, frontmatter metadata must be present (`path`, and optionally `type`)

File: README.md [hunk 3]
```diff
@@ -39,8 +46,19 @@ yarn               # or yarn install
...
+
+Cada ficheiro deve conter um _header_ com a meta-informação útil correspondente a cada ficheiro, principalmente:
+
+- `title: <título>`, onde `<título>` será o título que aparece associado à página correspondente ao ficheiro nos resumos;
+
+- `description: <bullet points>`, secção que deverá indicar sucintamente os pontos importantes abordados neste capítulo dos resumos (e que aparece no _embed_ do URL ao partilhar o link da página);
+
+- `path: /<UC>/<nome-da-pagina>`, _self-explanatory_;
+
+- `type: <categoria>`, tal como mencionado acima provavelmente apenas vos interessará a categoria `content`.
```
Comment (diogotcorreia) [inline README.md:61]: Maybe it'd be nice to put here a full list of available types (or maybe just in the in-depth docs in the wiki)

File: README.md [hunk 4]
```diff
@@ -50,8 +68,8 @@ yarn dev
 
 ### Formatar o código
 
-Antes de dar commit, é recomendado executar o `prettier` (se usarem um editor de texto - e.g. vscode - que já executa automaticamente, não é preciso).
+Antes de fazer um commit, é recomendado executar o `prettier` (se usarem um editor de texto - e.g. vscode - que já executa automaticamente, não é preciso). Devem correr o comando na raiz do _repo_ (`/home/.../resumos-leic`, portanto).
 
 ```bash
-yarn format
+yarn format:content
```
Comment (diogotcorreia) [inline README.md:74]: This is not necessary anymore, `yarn format` is an alias to that

**Review by diogotcorreia — CHANGES_REQUESTED** [review]
Also, it's probably a good idea to link the wiki here, since it's a more complete guide

File: README.md (hunk 4 above)
Comment (randomicecube) [reply README.md:74]: TIL, ty!

File: README.md (hunk 3 above)
Comment (randomicecube) [reply README.md:61]: I'd argue it'd be fine for it to be fully documented in the Wiki (and for us to include only the essentials here), but it's your call

**Comment by randomicecube** [comment]
> Also, it's probably a good idea to link the wiki here, since it's a more complete guide

This is now referenced at the end of the file (as a sort of foot note), tell me if you think it should be higher up

**Review by diogotcorreia — APPROVED** [review]

**Pull request merged**